README.in

```diff
@@ -16,39 +16,83 @@ Code is currently maintained on GitHub:
  REQUIREMENTS
 ==============
 
-rt-app runs on GNU/Linux. It needs autoconf, automake, libtool , libjson-c and
-a recent compiler (mainly: gcc) for basic features.
+rt-app runs on GNU/Linux. It needs bash, autoconf, automake, libtool,
+libjson-c, GNU make and a recent compiler (tested on: gcc) for basic features.
 
-=============
- COMPILATION
-=============
 
-$ autoreconf --install
-$ ./configure
-$ make
-$ make install
+=================
+ BUILDING json-c
+=================
 
-Last step is optional, rt-app is built in the src/ directory.
+If you are not happy using the version installed by your packaging system,
+if it does not provide static libraries and you need them, need to
+cross-compile, build it from source like this:
 
-Typical usage:
+retrieve source code available here:
 
-$ ./configure --prefix=<directory>
-	installs the compiled program in the given directory
+    https://github.com/json-c/json-c
 
-$ ./configure --with-deadline
-	builds rt-app with support for SCHED_DEADLINE
+cross-compile json-c and build both static and shared libraries for aarch64:
 
-See ./configure --help for additional options.
+    export ac_cv_func_malloc_0_nonnull=yes
+    export ac_cv_func_realloc_0_nonnull=yes
+    ./autogen.sh
+    ./configure --host=aarch64-linux-gnu --enable-shared --enable-static
+    make
+
+
+================================
+ BUILDING AND INSTALLING rt-app
+================================
+
+cross-compile a static rt-app for aarch64, using your own json-c build that
+hasn't installed (or not into the standard locations):
```

Comment:
isn't ?

Reply:
i meant the build (process) hasn't installed. but yes, sounds a bit weird...

how about "...build that *wasn't* installed"